models/refugee.py

```diff
@@ -0,0 +1,123 @@
+        CHRONICLUNG = "Chronic lung diseases"
+        DEMENTIA_OR_NEUROLOGICAL = "Dementia or other neurological conditions"
+        DIABETES = "Diabetes"
+        DOWNSYNDROME = "Down Syndrome"
+        HEART = "Heart conditions"
+        HIV = "HIV infection"
+        IMMUNOCOMPROMISED = "Immunocompromised state (weakened immune system)"
+        MENTALHEALTH = "Mental health conditions"
+        OVERWEIGHT_OR_OBESITY = "Overweight and obesity"
+        PREGNANCY = "Pregnancy"
+        SICKLECELL_OR_THALASSEMIA = "Sickle cell or thalassemia"
+        SMOKING = "Smoking, current or former"
+        TRANSPLANT = "Solid organ or blood stem cell transplant"
+        STROKE_OR_CEREBROVASCULAR = "Stroke or cerebrovascular disease, which affects blood flow to the brain"
+        SUBSTANCE_USE = "Substance use disorders"
+        TUBERCULOSIS = "Tuberculosis"
+        OTHERS = "Others"
+
+    def __init__(self,
+                 firstname: str,
+                 lastname:str,
+                 camp: str,
+                 numOfFamilyMember: int,
+                 startingDate:date,
+                 medicalConditionType=None):
+
+        """
+        :param firstname: firstname of refugee family
+        :param lastname: lastname of refugee family
+        :param camp: camp of the refugee family locating at
+        :param numOfFamilyMember: param number of family member
+        :param startingDate: start date of refugee family creation
+        :param medicalConditionType: medical condition type of the refugee
+        """
+        self.__sanitise_name(firstname, lastname)
+        self.firstname = firstname
+        self.lastname = lastname
+        self.numOfFamilyMember = self.__sanitise_num_of_family_member(numOfFamilyMember)
+        self.camp = camp
+        self.startingDate = self.__sanitise_starting_date(startingDate)
+        self.medicalConditionType : set[self.MedicalCondition] = self.__sanitise_medical_condition_type(medicalConditionType)
+
+    def __sanitise_name(self, firstname, lastname):
+        """
+        check if name is valid
+        """
+        if not isinstance(firstname, str) or not isinstance(lastname, str):
+            raise self.InvalidNameException()
+        elif not firstname.isalpha() or not lastname.isalpha():
+            raise self.InvalidNameException()
+
+
+    def __sanitise_num_of_family_member(self, numOfFamilyMember):
+        """
+        check if number of family member is valid
+        """
+        if not isinstance(numOfFamilyMember, int) or numOfFamilyMember < 1:
+            raise self.InvalidNumOfFamilyMemberException()
+        return numOfFamilyMember
+
+    def __sanitise_medical_condition_type(self, medicalConditionType):
+        """
+        check if set(medicalConditionType) is created if medicalConditionType is not None.
+        """
+        emptySet = set()
+        if medicalConditionType:
+            return set(medicalConditionType)
+        else:
+            return emptySet
```

Comment:
since you are using emptySet just once, you can just write
```python
else:
    return set()
```
Also, variables should be snake_case